src/popup/router/components/Amount.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div class="amount">
```

Comment:
Maybe `AmountInput` can better describe the purpose of the component?

Reply:
Is this wrapping div necessary?